src/Core/Framework/Test/DataAbstractionLayer/Search/EntitySearcherTest.php

```diff
@@ -90,6 +90,7 @@ public function testTotalCountWithSearchTerm(): void
 
         static::assertSame(2, $result->getTotal());
         static::assertCount(2, $result->getEntities());
+        static::assertSame(1, $result->getPage());
```

Comment:
Test case with page 2 would be nice 😅 . Can be also without using the DAL just directly construct that class with a matrix

Reply:
Implemented :)